src/Database/Adapter/PdoAdapter.php

```diff
@@ -403,6 +409,50 @@ private function isMulti(array $data): bool
         return true;
     }
 
+    private function splitColumnNameAndOperator($key, $value, &$columnName, &$operator): void
+    {
+        // initialize both column name and operator
+        $columnName = $key;
+        $operator = '=';
+
+        // check presence of operator in $key
+        if (preg_match('/^(.*)\s(=|!=|<>|<|<=|>|>=)$/', $key, $matches) === 1) {
```

Comment:
is there any reason to support any white space? isn't one single space ` ` enough?

Reply:
Actually no reason, probably it would be enough